docs/content/reference/concepts-and-personas.md

```diff
@@ -0,0 +1,45 @@
+# Concepts and Personas
+
+## Concepts
+
+**Management Cluster**
+
+An OpenShift cluster which hosts HyperShift and zero to many Hosted Clusters.
+
+**Hosted Cluster**
+
+An OpenShift API endpoint managed by HyperShift.
```

Comment:
Maybe rather something like "A custom resource representing a cluster managed by Hypershift" since you would e.G. say "my hostedcluster is running" and then not refer to the api but the object in the api?

Reply:
Note these defs are carry over from the original concepts file where the intent was to have a common language for "concepts", this might or might not map to an underlying CRD.
e.g A Hosted Cluster (concepts) is "An OpenShift API endpoint managed by HyperShift" (Result of applying a HostedCluster CR)